Vizzuality/landgriffon
# Table styles according the design

### General description

* Moved table component from containers to components folder
* We should be able to click on chevron icon to collapse and expand the table, currently we are able to click in all the row except the icon
* Missing a green border on the left when expanding the parent
* Fixed warnings related with index key prop

### Designs

https://www.figma.com/file/WxYMmJrUZAgV6f0Csmt8fK/LG-Tool---Work-in-progress?node-id=2321%3A44494

### Testing instructions

Inside analysis, browse to table mode.
Apart from the added feature, check overall performance, styling...

## Checklist before merging
